### PR TITLE
Some tidying in alpha prep stage

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

File: admin/includes/languages/english.php [hunk 1]
```diff
@@ -4,13 +4,15 @@
  * @copyright Copyright 2003-2016 Zen Cart Development Team
  * @copyright Portions Copyright 2003 osCommerce
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
- * @version $Id: Author: DrByte  Wed Mar 23 14:21:26 2016 -0500 Modified in v1.5.5 $
+ * @version english.php Modified in v1.5.6 $
```
Comment (zcwilt) [inline admin/includes/languages/english.php]: minor oops

Comment (drbyte) [inline admin/includes/languages/english.php]: Updated

**Comment by zcwilt** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Pull request merged**